CMakeLists.txt

```diff
@@ -127,6 +127,15 @@ IF(NOT WIN32)
   ELSE()
     MESSAGE(FATAL_ERROR "Driver Manager was not found")
   ENDIF()
+
+  # Looking for iconv files
+  INCLUDE(${CMAKE_SOURCE_DIR}/cmake/FindIconv.cmake)
+  IF(ICONV_FOUND)
+    INCLUDE_DIRECTORIES(${ICONV_INCLUDE_DIR})
+    SET(PLATFORM_DEPENDENCIES ${PLATFORM_DEPENDENCIES} ${ICONV_LIBRARIES})
```

Comment:
Perhaps it makes sense to add ICONV_LIBRARIES to dependencies only if ICONV_EXTERNAL. Or that is another reason to add IF(APPLE)